stdlib/imaplib.pyi

```diff
@@ -61,7 +61,7 @@ class IMAP4:
     def socket(self) -> _socket: ...
     def recent(self) -> _CommandResults: ...
     def response(self, code: str) -> _CommandResults: ...
-    def append(self, mailbox: str, flags: str, date_time: str, message: ReadableBuffer) -> str: ...
+    def append(self, mailbox: str, flags: str, date_time: str, message: ReadableBuffer) -> tuple[str, list[bytes]]: ...
```

Comment:
An alias is used for `builtins.list` in this file to avoid naming conflict with `list` method
```suggestion
    def append(self, mailbox: str, flags: str, date_time: str, message: ReadableBuffer) -> tuple[str, _list[bytes]]: ...
```